lib/meandro/rules/unused_struct_field.ex

```diff
@@ -0,0 +1,149 @@
+defmodule Meandro.Rule.UnusedStructField do
+  @moduledoc """
+  Finds struct fields that are not used.
+  It has the following assumptions:
+    - As we are not tracking the value of each variable and a variable can be
+      reassigned we don't check the struct name when looking for access to a
+      struct field or modification
+  """
+
+  @behaviour Meandro.Rule
+
+  @impl true
+  def analyze(files_and_asts, _options) do
+    List.flatten(
+      for {file, ast} <- files_and_asts,
+          result <- analyze_module(file, ast, files_and_asts) do
+        result
+      end
+    )
+  end
+
+  @impl true
+  def is_ignored?(module, module) do
```

Comment:
```suggestion
  def is_ignored?(field, field) do
```
(see my comment about patterns below)

Reply:
Also keep in mind that if people want to ignore a rule for the *whole module*, that doesn't go through this function.

Reply:
Added proper pattern: {struct_name, field_name}